supernode/daemon/mgr/gc/gc_peer.go

```diff
@@ -40,7 +40,7 @@ func (gcm *Manager) gcPeers(ctx context.Context) {
 			continue
 		}
 
-		if peerState.ServiceDownTime != 0 &&
+		if peerState.ServiceDownTime == 0 ||
 			timeutils.GetCurrentTimeMillis()-peerState.ServiceDownTime < int64(gcm.cfg.PeerGCDelay) {
```

Comment:
Actually I like your former way to add a boolean to describe `timeutils.GetCurrentTimeMillis()-peerState.ServiceDownTime < int64(gcm.cfg.PeerGCDelay)`. However, I do not think `interval` is readable. We should use a more readable variable name, like `xxxxxxInterval`.

Reply:
I remove the former way because if` peerState.ServiceDownTime==0`,supernode can go on instead of calculating `timeutils.GetCurrentTimeMillis()-peerState.ServiceDownTime` again